documentation/asciidoc/computers/configuration/display-resolution.adoc

```diff
@@ -18,69 +18,87 @@ Flagship models since Raspberry Pi 4B, Compute Modules since CM4 (except for CM4
 
 === Set resolution and rotation
 
-On the Raspberry Pi Desktop, open the *Preferences* menu and select the **Screen Configuration** utility. You should see a graphical representation of the displays connected to the Raspberry Pi. Right click on the display you wish to modify, and select an option. Click **Apply** to and close **Screen Configuration** to save your changes.
+In Raspberry Pi OS Trixie, display resolution and screen rotation are managed using the 'Control Centre' application.  
+This replaces older tools such as 'Screen Configuration', 'raindrop', and 'arandr', which were used in previous Raspberry Pi OS releases.
 
-Alternatively, use the following command to open the **Screen Configuration** utility:
+If you are following older guides or tutorials, note that those tools are no longer present or required.
 
-[source,console]
-----
-$ raindrop
-----
+=== Change resolution and rotation using the graphical interface (recommended)
 
-[TIP]
-====
-If your installation of Raspberry Pi OS doesn't already include `raindrop`, you can install it with the following command:
+This is the easiest and recommended method for most users.
 
-[source,console]
-----
-$ sudo apt install raindrop
-----
+STEPS
 
-Older versions of Raspberry Pi OS used a different screen configuration utility named `arandr`. To uninstall `arandr`, run the following command:
+1. Start your Raspberry Pi and log in to the desktop.
+2. Look at the **top-right corner of the screen** to find the system tray.
+3. Click on the 'Control Centre' icon.
```

Comment:
Hmmm - the "Control Centre" is available from the "Preferences" sub-menu of the main applications menu, I don't think that it's available from the system tray?